openerp/service/server.py

```diff
@@ -232,7 +232,12 @@ def close_socket(self, sock):
         sock.close()
 
 class ThreadedServer(CommonServer):
+    initial_threads = []
+    
     def __init__(self, app):
+        for t in threading.enumerate():
+            self.initial_threads.append(t)
```

Comment:
> so it will only skip threads in the list

Ah yes, my bad, I'd misunderstood that part.

Either way, I still don't think that's the correct fix.